packages/near-fast-auth-signer/src/lib/useAuthState.ts

```diff
@@ -63,8 +71,31 @@ export const useAuthState = (skipGetKeys = false): AuthState => {
             setAuthenticated(true);
           }
         }).catch(() => setAuthenticated(false));
+    } else {
+      checkFirestoreReady().then(async (isReady) => {
+        if (isReady) {
+          // @ts-ignore
+          const oidcToken = firebaseAuth.currentUser.accessToken;
+          if (window.fastAuthController.getLocalStoreKey(`oidc_keypair_${oidcToken}`)) {
+            const recoveryPK = await window.fastAuthController.getUserCredential(oidcToken);
+            const accountIds = await fetch(`${network.fastAuth.authHelperUrl}/publicKey/${recoveryPK}/accounts`)
+              .then((res) => res.json())
+              .catch((err) => {
+                console.log(err);
+                throw new Error('Unable to retrieve account Id');
+              });
+            (window as any).fastAuthController = new FastAuthController({
```

Comment:
Do we need the any here? I guess the fastAuthController it's defined on the global ts file, no?